bin/improver-tests

```diff
@@ -18,46 +18,94 @@ function echo_ok {
     echo -e "\033[1;32m[OK]\033[0m $1"
 }
 
-if [[ ${1:-} == '--help' ]] || [[ ${1:-} == '-h' ]]; then
+function improver_test_pep8 {
+    # PEP8 testing.
+    ${PEP8:-pep8} improver
+    echo_ok "pep8"
+}
+
+function improver_test_pylint {
+    # Pylint score generation.
+    ${PYLINT:-pylint} --rcfile=../etc/pylintrc improver
+}
+
+function improver_test_pylintE {
+    # Pylint obvious-errors-only testing.
+    ${PYLINT:-pylint} -E --rcfile=../etc/pylintrc improver
+    echo_ok "pylint -E"
+}
+
+function improver_test_unit {
+    # Unit tests.
+    python -m unittest discover
+    echo_ok "Unit tests"
+}
+
+function improver_test_cli {
+    # CLI testing.
+    PATH="$IMPROVER_DIR/tests/bin/:$PATH"
+    if [[ ${1:-} != '--debug' ]] && type prove &>/dev/null; then
```

Comment:
Fair point, I have changed the meaning of `--debug` to run `prove -v` rather than `prove` so that the meaning is purer. Pure `bats` doesn't have a verbose option.